backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/internal/BpkCalendarDayCell.kt

```diff
@@ -70,15 +71,20 @@ internal fun BpkCalendarDayCell(
         horizontalAlignment = Alignment.CenterHorizontally,
         modifier = modifier
             .padding(bottom = BpkSpacing.Lg)
-            .selectable(
+            .clickable(
                 indication = null,
-                selected = selection != null,
+//                selected = selection != null,
```

Comment:
Sorry I'm not sure if I understand the first sentence,  what do you mean by not having the accessibility labels?

About `selected`, yes it was initially included because Gert and I made a copy of `modifier.selectable`, because the original one didn't have `onClickLabel` and we needed it. In the end though, and after some changes, we noticed that `selected` was not used anywhere and we could use `clickable` here instead and remove `selected`. I don't know if that helps, I might be missing some details? Do you think `selected` should not be removed? 
Also sorry about accidentally leaving comments in the code. :)

Reply:
sorry what i mean is that in some cases we won't have an accessibility label (either because there's nothing provided by the consumer or this exact mode doesn't have custom labels). In this case we should still set `selected` to make sure we're still getting the default selected behaviour even without accessibility labels set (`onClickLabel` is nullable, so we still need to support the null case)

Reply:
Thanks for the comments! I've changed it so that we're using selectable again.